appengine/standard/flask/tutorial/requirements.txt

```diff
@@ -1 +1,2 @@
+Werkzeug==0.12.2
```

Comment:
To prevent our auto-updating script from messing with this, can you change it to a range? (& also add a note as to why we pin it)

```Werkzeug<=0.13```